frontend/src/scenes/web-analytics/tiles/WebAnalyticsTile.tsx

```diff
@@ -536,47 +536,9 @@ export const WebStatsTableTile = ({
         }
     }, [onClick, insightProps])
 
-    const pathCleaningSettingsUrl = urls.settings('project-product-analytics', 'path-cleaning')
     return (
         <div className="border rounded bg-bg-light flex-1 flex flex-col">
-            {showPathCleaningControls && (
-                <div className="flex flex-row items-center justify-end m-2 mr-4">
-                    <div className="flex flex-row items-center space-x-2">
-                        <LemonSwitch
-                            label={
-                                <div className="flex flex-row space-x-2">
-                                    <Tooltip
-                                        title={
-                                            <>
-                                                Check{' '}
-                                                <Link to="https://posthog.com/docs/product-analytics/paths#path-cleaning-rules">
-                                                    our path cleaning rules documentation
-                                                </Link>{' '}
-                                                to learn more about path cleaning
-                                            </>
-                                        }
-                                        interactive
-                                    >
-                                        <span>Enable path cleaning</span>
-                                    </Tooltip>
-                                    <LemonButton
-                                        icon={<IconGear />}
-                                        type="tertiary"
-                                        status="alt"
-                                        size="small"
-                                        noPadding={true}
-                                        tooltip="Edit path cleaning settings"
-                                        to={pathCleaningSettingsUrl}
-                                    />
-                                </div>
-                            }
-                            checked={!!isPathCleaningEnabled}
-                            onChange={setIsPathCleaningEnabled}
-                            className="h-full"
-                        />
-                    </div>
-                </div>
-            )}
+            {control != null && <div className="flex flex-row items-center justify-end m-2 mr-4">{control}</div>}
```

Comment:
The types allow `control` to be `undefined`. Do we want to render the div in that case? I believe we shouldn't

```suggestion
            {control && <div className="flex flex-row items-center justify-end m-2 mr-4">{control}</div>}
```

Reply:
This is a JavaScript quirk

`control != null`
is equivalent to
`control !== null && control !== undefined`

As an aside, I try to avoid the second form (this one: `{value && <Component/>}`), as falsey values of component can leak through (what would happen if `value` was `0`?). You would want to use something like `{!!value && <Component/>}`, as react won't render a boolean value.

Reply:
> This is a JavaScript quirk
> 
> `control != null` is equivalent to `control !== null && control !== undefined`

I always forget that, thanks 🤦🏻

> You would want to use something like {!!value && <Component/>}, as react won't render a boolean value.

100% if `value` can be a number, but the types here wouldn't allow this :)